### PR TITLE
Add redux add-book and remove-book func

### PR DESCRIPTION
**Pull request by Paulinelali**
In this PR, I:

 - [x] Display books, received from the slice, in a reusable component
  - [x] Dispatch actions using useDispatch
  - [x] An event handler that adds a book to the books array
  - [x] An event handler that removes a book from the books array (by id)

### CODE REVIEW

File: src/Pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { BookList, Form } from '../components';
+
```
Comment (TadesseDev) [inline src/Pages/Books.js:3]: - [ ] Dear @Paulinelali, Though you did well implementing the functionalities, I'm experiencing an issue with the imports. I notice you have used Uppercase for your imports and `all small` in your file names. As you can see here your form is named `form.js` but you are importing it as `Form`. This seems to be causing the problem. Also, this same issue happened for the navbar and other components. Please take a look and fix the issues.

### Issues
Screenshot from 2023-07-21 09-58-44
- [ ] After fixing the issues, Please make sure all functionalities are working as expected.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Paulinelali 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x]  Most of the requirement has been meet. ✅
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Pages/Books.js (hunk 1 above)
Comment (Paulinelali) [reply src/Pages/Books.js:3]: ### Please can you give 5 minutes of your time so we meet on call?
I can't see this error on my screen here and also, the linter checked all errors and non was found, so I'm kind of confused.
please let me know so I can send you an invite
Thank your

Comment (TadesseDev) [reply src/Pages/Books.js:3]: Dear @Paulinelali, there is a case sensitivity issue (This is usually check when working with a Linux device). As I stated, Please use the same case for the file name and the import. This should solve the problem.

**Pull request closed without merging**